### PR TITLE
Move config to beamlime.config.raw_detectors for creating raw views

### PR DESCRIPTION
**Pull request by MridulS**
This PR moves the config of detectors to `beamlime.config.raw_detectors` and cleans up the `handlers.py` a bit. For now these are just python files but we can make them into TOML/YAML/.... in the future. We can also move them to instrument packages and register them via entry-points. I chose `beamlime.config.raw_detectors` to be explicit that this is the configuration used during raw detector view.

### CODE REVIEW

File: src/beamlime/config/raw_detectors/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+This config is used during Live Data Reduction detector view.
+Currently the instrument specific config is stored in python files, but
+they can be moved to a separate file format in the future.
+"""
+
+from .dream import _dream
+from .loki import _loki
+from .nmx import _nmx
+
+__all__ = ["_dream", "_loki", "_nmx"]
```
Comment (SimonHeybrock) [inline src/beamlime/config/raw_detectors/__init__.py:11]: Should we consider it public now and remove the leading `_`?

File: src/beamlime/config/raw_detectors/dream.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Order in 'resolution' matters so plots have X as horizontal axis and Y as vertical.
```
Comment (SimonHeybrock) [inline src/beamlime/config/raw_detectors/dream.py:1]: License headers missing?

File: src/beamlime/config/raw_detectors/nmx.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        'Panel 2': {
+            'detector_name': 'detector_panel_2',
+            'gridspec': (0, 2),
+            'projection': raw.LogicalView(),
+        },
+    },
+}
+# These banks currently have out-of-range event_id values, so we ignore them.
+del _nmx['detectors']['Panel 1']
+del _nmx['detectors']['Panel 2']
```
Comment (SimonHeybrock) [inline src/beamlime/config/raw_detectors/nmx.py:25]: I think this was fixed now, both in CODA files as well as `ess.reduce.live.raw` now discarding out-of-range IDs. I think it is safe to remove these lines.

File: src/beamlime/config/raw_detectors/__init__.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+"""
+This config is used during Live Data Reduction detector view.
```
Comment (SimonHeybrock) [inline src/beamlime/config/raw_detectors/__init__.py:2]: Not data-reduction actually, how about something like
```suggestion
This config is used to setup live raw detector views.
```

File: src/beamlime/config/raw_detectors/__init__.py (hunk 1 above)
Comment (MridulS) [reply src/beamlime/config/raw_detectors/__init__.py:11]: I would then argue to make it something like `dream_detectors_config`, so we don't litter around a python object `dream` in the python namespace.

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**